src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -321,7 +322,7 @@ private void initKeyBindings() {
                         event.consume();
                         break;
                     case FOCUS_GROUP_LIST:
-                        this.stateManager.getGroupTree().requestFocus();
+                        getGroupTreeView().getGroupTree().requestFocus();
```

Comment:
Instead of exposing the whole API of the tree view I would suggest just to add a request focus method to the API of the sidepane component, which itself then calls the requestfocus of it's child.

Reply:
added request focus method in sidepane component